spec/controllers/voucher_adjustments_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe VoucherAdjustmentsController, type: :controller do
```

Comment:
I was going to say we should move this to a request spec, but I see Gaetan already has 👍